lib/workos/types/verify_factor_struct.rb

```diff
@@ -7,9 +7,9 @@ module Types
     # for the Factor class
     class VerifyFactorStruct < T::Struct
       const :challenge, T.nilable(T::Hash[Symbol, Object])
-      const :valid, T.nilable(TrueClass)
+      const :valid, T.any(TrueClass, FalseClass)
       const :code, T.nilable(String)
       const :message, T.nilable(String)
     end
   end
-end
+end
```

Comment:
We'll need to add this newline back to remove the Robocop warning that's failing CI.

Reply:
done!